docs/AboutUs.md

```diff
@@ -9,51 +9,39 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Drustan Yeo 
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/drustanyjt.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
-
-* Role: Project Advisor
-
-### Jane Doe
-
-<img src="images/johndoe.png" width="200px">
-
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/drustanyjt)]
+[[linkedin](https://www.linkedin.com/in/drustan/)]
 
 * Role: Team Lead
-* Responsibilities: UI
 
-### Johnny Doe
+### Jayne Chen 
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/jayne1010.png" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](http://github.com/Jayne1010)]
 
 * Role: Developer
-* Responsibilities: Data
+* Responsibilities: Documentation
 
-### Jean Doe
+### Song Jia Hui
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/jiahui0309.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/jiahui0309)] 
+[[linkedin](https://www.linkedin.com/in/song-jia-hui/)]
 
 * Role: Developer
-* Responsibilities: Dev Ops + Threading
+* Responsibilities: Integration
 
-### James Doe
+### Charmaine Teng Shi Rou
```

Comment:
Can change the order of my name to Teng Shi Rou Charmaine! And on second thought can you also add my linked in (this is the link: www.linkedin.com/in/charmaine-teng) Thanks :)